ParabolInc/parabol
# feat: change email domain

Fix https://github.com/ParabolInc/parabol/issues/5921

To test:

Run the following mutation
```gql
mutation {
  changeEmailDomain(oldDomain: "parabol.co", newDomain: "parabol-team.com"){
    ... on ChangeEmailDomainSuccess {
      users {
        email
      }
    }
  }
}
```
See that the email address or domain is updated in the following tables in RethinkDB:

- [ ] Organization
- [ ] TeamMember
- [ ] SAML
- [ ] Invoice, update the billingLeaderEmails field

In Postgres:

- [ ] OrganizationApprovedDomain
- [ ] User
